docs/source/howto/data_manage.rst

```diff
@@ -0,0 +1,41 @@
+.. _how-to:data-manage:
+
+*****************************
+How to import and manage data
+*****************************
+
+
+.. _how-to:data-manage:load-external:
+
+How to load external data
+=========================
+
+
+.. _how-to:data-manage:plugins:
+
+How to create custom data types
```

Comment:
I would put this at the end of the section, as this is advanced (for "plugin developers"?).

Reply:
Fine to place it towards the end. Just maybe in the future, once people from other domains start to use AiIDA, the need to create custom data types becomes more common. Anyway, there is a link at the end of the section of using existing types, so it is fine to move to the end for now